eng/common/templates/steps/run-build-from-source.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  portableOnly: 'false'
+
+steps:
+  - script: |
```

Comment:
Unnecessary indent level, current style seems to be only indent when needed.

Reply:
Fixed